assets/scss/modules/_accordion.scss

```diff
@@ -31,16 +31,16 @@
         height: 7px;
         content: '';
         left: 0;
-        top: 29px;
-        border-left: solid 6px $dp-color-grey;
-        border-bottom: solid 6px transparent;
-        border-top: solid 6px transparent;
+        top: 31px;
```

Comment:
What is the reason why is used this kind of numbers? I mean, why not `30px`?

Reply:
In this case it is a pseudo class for an "icon" in an absolute position, so we can move it to the pixel. In this case we align it to the content. The normalization of spaces that we use in the library is 6px and its multiples, only for paddings and margins. Does not apply in this case.